doc/html/hummingbird/ml/convert.html

```diff
@@ -26,7 +26,7 @@ <h1 class="title">Module <code>hummingbird.ml.convert</code></h1>
 <details class="source">
 <summary>
 <span>Expand source code</span>
-<a href="https://github.com/microsoft/hummingbird/blob/master/hummingbird/ml/convert.py#L0-L271" class="git-link">Browse git</a>
```

Comment:
I think you need to reinstall your precommit hooks; these HTML files should no longer be getting added to PRs.

If you re-run `pre-commit install` it should work; else delete them from `.git/hooks/`

Reply:
Yea I think I was still using the version before you removed `pdoc3` from the pipeline.